src/github/pullRequestOverview.ts

```diff
@@ -224,6 +229,10 @@ export class PullRequestOverviewPanel {
 				return this.checkoutDefaultBranch(message.branch);
 			case 'pr.comment':
 				return this.createComment(message.text);
+			case 'scroll': {
+				this._scrollPosition = message.scrollPosition
```

Comment:
Missing semicolon :)

The linter should run when you do `yarn run watch`, but it's often the top of the output so easy to miss